### PR TITLE
[Element 2.0] Fix test that can not show image on iTerm2 when folder has the element config file

### PR DESCRIPTION
**Pull request by honglax**
Resolve issue [FLO-2172](https://tricentisflood.atlassian.net/browse/FLO-2172).

### CODE REVIEW

File: packages/core/src/ElementOption.ts [hunk 1]
```diff
@@ -246,13 +246,7 @@ export function normalizeElementOptions(
...
 		if (loopCount !== undefined) opts.testSettingOverrides.loopCount = loopCount
 		if (duration !== undefined) opts.testSettingOverrides.duration = duration
-		opts.headless = runArgs.headless ?? opts.headless
-		opts.devtools = runArgs.devtools ?? opts.devtools
-		opts.sandbox = runArgs.sandbox ?? opts.sandbox
-		opts.export = runArgs.export ?? opts.export
-		opts.browser = runArgs.browser ?? opts.browser
-		opts.executablePath = runArgs.executablePath ?? opts.executablePath
-		opts.downloadsPath = runArgs.downloadsPath ?? opts.downloadsPath
+		opts = { ...opts, ...runArgs }
```
Comment (HieuLeKMS) [inline packages/core/src/ElementOption.ts:249]: With this change, if `runArgs` doesn't have one of these properties, for example `headless`, then `opts.headless` won't be changed, but if `runArgs` does have the property `headless`, but its value is `undefined`, then `opts.headless` will be set to `undefined`, whereas with the old code, `opts.headless` won't be changed in both case.

Comment (honglax) [reply packages/core/src/ElementOption.ts:249]: If the end-user does not provide any arguments, the runArgs does not have that property. And this scope is executed when there is config file in the folder. 

If you and a @phileba wanna change it back to the old one and add a new line to check showScreenshot, I’m happy to do it.

**Review by nick-flood — APPROVED** [review]

File: packages/core/src/ElementOption.ts (hunk 1 above)
Comment (HieuLeKMS) [reply packages/core/src/ElementOption.ts:249]: If you are aware and sure there're no problems with it, then no need to change back

**Review by HieuLeKMS — APPROVED** [review]

**Review by phileba — APPROVED** [review]

File: packages/core/src/ElementOption.ts (hunk 1 above)
Comment (honglax) [reply packages/core/src/ElementOption.ts:249]: Got it.

**Pull request merged**